Packs/MicrosoftGraphMail/ReleaseNotes/1_6_20.md

```diff
@@ -0,0 +1,18 @@
+
+#### Integrations
+
+##### Microsoft Graph Mail Single User
+
+Fixed an issue where the following commands failed to decode the **htmlBody** argument if it was provided with no padding:
```

Comment:
Same as above:
You fixed also comments that do not use the htmlBody such as file_result_creator inside the create_attachment command (also would suggest changing the get-attachments to the real name), maybe i would change the release notes to :
Fixed an issue where the following commands failed to decode Base64-encoded attachments or images inside the htmlBody when provided without padding:

send-mail
reply-mail
msgraph-mail-get-attachment
fetch-incidents
wdyt?